new-site/css/style.css

```diff
@@ -245,3 +245,242 @@ textarea {
   }
 }
 
+/************************************************************************************
+MEDIA QUERIES
+*************************************************************************************/
+
+/* logo */
+#logo {
+  width: 100px;
+  height: 70px;
+  float: left;
+  color: white;
+  font-size: 100%;
+}
+
+
+/* navbar */
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1.5rem 7%;
+  background: rgb(50, 112, 170);
+  border-bottom: solid 1px rgba(255, 255, 255, 0.247);
+  position: sticky;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1000;
+}
+
+.Navbar li a {
+  padding: 8px;
+  justify-content: flex-end;
+  width: 100%;
+  display: flex;
+  font-size: 20px;
+  font-family: sans-serif;
+}
+
+.nav-bar ul {
+  margin: 0;
+  padding: 0;
+  display: inline;
+  color: #007bff;
+}
+
+.nav-bar li a {
+  display: block;
+  color: rgb(156, 83, 83);
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.nav-bar a {
+  text-decoration: none;
+  color: #333;
+  font-weight: bold;
+  font-size: 16px;
+}
+.nav-bar li {
+  float: right;
+  display: inline;
+  color: black;
+  margin: 0 15px;
+  border-radius: 10%;
+}
+
+.Navbar a:hover {
+  cursor: pointer;
```

Comment:
A link should normally have `cursor: pointer` anyway, so it's worth looking at why the cursor isn't changing, or removing this if it's not needed.